astropy/table/table.py

```diff
@@ -2021,6 +2021,70 @@ def group_by(self, keys):
 
         return groups.table_group_by(self, keys)
 
+    def to_pandas(self):
+        """
+        Return a :class:`pandas.DataFrame` instance
+
+        Returns
+        -------
+        dataframe : :class:`pandas.DataFrame`
+            A pandas :class:`pandas.DataFrame` instance
+        """
+        from pandas import DataFrame
+
+        if self.has_mixin_columns:
+            raise ValueError("Cannot convert a table with mixin columns to a pandas DataFrame")
+
+        for column in self.columns:
```

Comment:
Definitely no big deal.  Though since every usage of the `dtype` property recreates the dtypes of all the columns, maybe write:
```
if any(getattr(col, 'ndim', 1) > 1 for col in self.columns.values()):
    raise ...
```